packages/plugin-devtools/src/components/InfoComponent.ts

```diff
@@ -2,15 +2,19 @@ import { OwlUIComponent } from '../../../owl-ui/src/OwlUIComponent';
 import { VRange } from '../../../core/src/VRange';
 import { VNode } from '../../../core/src/VNodes/VNode';
 
+interface InfoValue {
+    range: VRange;
+}
 interface InfoState {
     currentTab: string;
-    range: VRange;
 }
 const vNodeKeys = Object.keys(new VNode('default'));
 export class InfoComponent extends OwlUIComponent<{}> {
+    values: InfoValue = {
+        range: this.env.editor.vDocument.range,
+    };
```

Comment:
@Zinston Couldn't these "values" be props ? We already have `state` and `props`, so I'm asking to avoid introducing yet another place to find the data.

Reply:
This is @Gorash 's commit, cherry-picked from another branch.